sources/packages/backend/apps/workers/src/controllers/supporting-user/supporting-user.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { SupportingUserType } from "@sims/sims-db";
+
+export interface CreateSupportingUsersJobInDTO {
+  applicationId: number;
+  supportingUsersTypes: SupportingUserType[];
+}
+
+export interface CreateSupportingUsersJobOutDTO {
+  createdSupportingUsersIds: number[];
```

Comment:
the property name can be supportingUserIds as the class name is CreateSupportingUsersJobOutDTO.

Reply:
It can make sense but these "OutDTOs" properties are going to be variables in Camunda and the class will not be part of it. When we look at class name+property your suggestion makes sense but from Camunda's perspective, we must have the property names self-explained. Does it make sense?